rotkehlchen/accounting/structures/processed_event.py

```diff
@@ -45,7 +45,7 @@ class ProcessedAccountingEvent:
     count_cost_basis_pnl: bool = field(init=False, default=False)
 
     def to_string(self, ts_converter: Callable[[Timestamp], str]) -> str:
-        desc = f'{self.type.name} for {self.free_amount}/{self.taxable_amount} {self.asset.resolve_to_asset_with_symbol().symbol} with price: {self.price} and PNL: {self.pnl}.'  # noqa: E501
+        desc = f'{self.type.name} for {self.free_amount}/{self.taxable_amount} {str(self.asset)} with price: {self.price} and PNL: {self.pnl}.'  # noqa: E501
```

Comment:
This is not okay. This is for the csv export. Can't have identifiers there. Identifiers are not human readable

Reply:
Could you point to a line please? I found it being used only in one case https://github.com/nebolax/rotki/blob/a9cf0eedfde3fa0b28202a8367715a8e0ef644f0/rotkehlchen/accounting/pot.py#L77